### ISSUE TITLE
Docs: old snippet could be removed.

### ISSUE THREAD

**Issue by guettli** [shtalinberg/django-el-pagination#28]
From http://django-el-pagination.readthedocs.io/en/latest/generic_views.html
```
NOTE: Django < 1.8 use this rule for urlpatterns:

from django.conf.urls.defaults import *
from django.views.generic import ListView
from books.models import Publisher

urlpatterns = patterns('',
    (r'^publishers/$', ListView.as_view(model=Publisher)),
)
```

I think it is useless  to show the old code here:

1. Django < 1.8 is old
1. The code snippet is only there to show what the code looks like **before** transforming to AjaxListView.

Why not remove the snippet from above from the docs?

**Comment by shtalinberg** [comment]
will be fixed in 3.0.0

**Comment by guettli** [comment]
thank you!
